src/components/index.js

```diff
@@ -1,3 +1,9 @@
+// Default export of all components first level of deep
+export { default as BarSelectionBrands } from './BarSelectionBrands/BarSelectionBrands';
```

Comment:
```suggestion
export { default as BarSelectionBrands } from './BarSelectionBrands';
```
add `index.js` on `./BarSelectionBrands` folder